core/unit_test/CMakeLists.txt

```diff
@@ -168,6 +168,7 @@ if(Kokkos_ENABLE_OPENMPTARGET)
   )
 endif()
 
+
```

Comment:
This change seems like unnecessary noise in the pull request. Please remove it (in the future, do a `git commit -p` when preparing a commit for a pull request so that you leave out noise like this in the diff).

Reply:
Resolved with ca4a458